src/main/java/core/basesyntax/service/Service.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface Service<T, I> {
```

Comment:
```suggestion
public interface GenericService<T, I> {
```
Let's use the `GenericService` name. It will be a bit clearer than just a `Service`.

Reply:
Fixed